app/helpers/bulk_updates_helper.rb

```diff
@@ -2,33 +2,128 @@ module BulkUpdatesHelper
-    ['Data Type', 'data_type'],
-    ['Downloadable', 'downloadable'], # * should be an enum (true / false)
-    ['ECHO Collection ID', 'echo_collection_id'],
-    ['Entry ID', 'entry_id'],
-    ['Entry Title', 'entry_title'],
-    ['Has Granules', 'has_granules'], # * should be an enum (true / false)
-    ['Instrument', 'instrument'],
-    ['Online Only', 'online_only'], # * should be an enum (true / false)
-    ['Platform', 'platform'],
-    ['Processing Level ID', 'processing_level_id'],
-    ['Project', 'project'],
-    ['Revision Date', 'revision_date'], # * should be a date picker
-    ['Science Keywords', 'science_keywords'],
-    ['Sensor', 'sensor'],
-    ['Short Name', 'short_name'],
-    ['Spatial Keyword', 'spatial_keyword'],
-    ['Temporal', 'temporal'],
-    ['Updated Since', 'updated_since'], # * should be a date picker
-    ['Version', 'version']
-  ].freeze
+  SEARCHABLE_KEYS = {
+    'two_d_coordinate_system_name': {
+      title: 'Tiling Identification System Name',
+      data_attributes: {
+        supports_wildcard: false,
+        format: 'text'
+      }
+    },
+    'concept_id': {
+      title: 'Concept ID',
+      data_attributes: {
+        supports_wildcard: false,
+        format: 'text'
+      }
+    },
+    'doi': {
+      title: 'DOI Value',
+      data_attributes: {
+        supports_wildcard: false,
+        format: 'text'
+      }
+    },
+    'data_center': {
+      title: 'Data Center',
+      data_attributes: {
+        supports_wildcard: true,
+        format: 'text'
+      }
+    },
+    'collection_data_type': {
+      title: 'Collection Data Type',
+      data_attributes: {
+        supports_wildcard: false,
+        format: 'text'
+      }
+    },
+    'entry_title': {
+      title: 'Entry Title',
+      data_attributes: {
+        supports_wildcard: true,
+        format: 'text'
+      }
+    },
+    'instrument': {
+      title: 'Instrument',
+      data_attributes: {
+        supports_wildcard: true,
+        format: 'text'
+      }
+    },
+    'platform': {
+      title: 'Platform',
+      data_attributes: {
+        supports_wildcard: true,
+        format: 'text'
+      }
+    },
+    'processing_level_id': {
+      title: 'Processing Level ID',
+      data_attributes: {
+        supports_wildcard: true,
+        format: 'text'
+      }
+    },
+    'project': {
+      title: 'Project',
+      data_attributes: {
+        supports_wildcard: true,
+        format: 'text'
+      }
+    },
+    'revision_date': {
+      title: 'Revision Date',
+      data_attributes: {
+        supports_wildcard: false,
+        format: 'date',
+        description: 'Accepted Date Format: yyyy-mm-ddThh:mm:ss.SSSZ'
+      }
+    },
+    'science_keywords': {
+      title: 'Science Keywords',
+      data_attributes: {
+        supports_wildcard: false,
+        format: 'text',
+        description: 'Accepted Values: a value at any level of the Science Keyword hierarchy'
```

Comment:
'Searches for provided search term at all levels of the science keyword hierarchy.'